src/executables/master_heartbeat.c

```diff
@@ -2687,7 +2689,17 @@ hb_cluster_load_group_and_node_list (char *ha_node_list, char *ha_replica_list)
 	    {
 	      if (are_hostnames_equal (node->host_name, hb_Cluster->host_name))
 		{
-		  hb_Cluster->myself = node;
+		  if (hb_Cluster->state == HB_NSTATE_REPLICA)
+		    {
+		      sprintf (err_string, "In replica mode, (%s) must not be in the %s", hb_Cluster->host_name,
```

Comment:
Nit.
```suggestion
		      sprintf (err_string, "The replica node(%s) must not be specified in the %s", hb_Cluster->host_name,
```

Reply:
thank you.
Adding "specified" word is good. 
But, I think it would be clearer to use the words "in replication mode" to let the user know which mode was set in the error message.